manifests/kustomize/env/gcp/ml-pipeline-apiserver-deployment-patch.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: ml-pipeline
+spec:
+  template:
+    spec:
+      containers:
+        - name: ml-pipeline-api-server
+          env:
+            - name: OBJECTSTORECONFIG_BUCKETNAME
+              # Replace with your own bucket name
+              value: 'yang-experiment-6-mlpipeline'
```

Comment:
Should this be replaced with a null or better value?

Reply:
thanks. I'll do a clean up in a separate PR soon to clean up the configuration.